### ISSUE TITLE
Add a pre-commit hook for jupytext

### ISSUE THREAD

**Issue by gtca** [scverse/cookiecutter-scverse#19]
After #9 and #12 are merged.
See https://jupytext.readthedocs.io/en/latest/using-pre-commit.html.

**Comment by grst** [comment]
What would be the purpose? I just `.gitignore` the ipynb files and jupytext anyway syncs them every time a notebook is saved.

**Comment by gtca** [comment]
[scanpy][scanpy-tutorials], [muon][muon-tutorials], [scvi-tools][scvi-tutorials] all use rendered `.ipynb` notebooks. As I understand, we want to keep this opportunity, especially for the large workflows, where we don't want to or can't rerun them regularly. So we can't just _gitignore_ them?

The idea behind this was, I believe, to also have rendered `.ipynb` files. Or maybe just providing the guidance for setting up `jupytext` might be enough. I guess you could lead the best practices here if you're already using it.
Also, don't we need to sync `.md` and `.ipynb` files also in case `.md` files were modified? I can see how this is not an issue if `.ipynb` files are ignored.

[muon-tutorials]: https://github.com/scverse/scanpy-tutorials
[scanpy-tutorials]: https://github.com/scverse/scanpy-tutorials
[scvi-tutorials]: https://github.com/scverse/scvi-tutorials

**Comment by grst** [comment]
I understand why in some cases the `ipynb` files are needed, but in that case why not *just* add the ipynb files and not bother with jupytext at all? 

It could be nice for code-reviews though...

**Comment by grst** [comment]
Best to implement this together with #40.

**Comment by grst** [comment]
Closing in favor of #40
